### ISSUE TITLE
hc22000 dumped in current directory and not in destination specified with -o

### ISSUE THREAD

**Issue by ButtBundy** [Ragnt/AngryOxide#49]
AngryOxide 0.8.28

hc22000 files are not dumped in the directory set with -o but in the directory from which the command is executed

**Comment by Ragnt** [comment]
good call, I need to re-implement -o, I may add a new argument for specifying a directory (so -o will just be filename) and also need to probably differentiate between -o and -O (one overwrites, the other adds a new file).

Thanks.
